pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-searching_modules.adoc

```diff
@@ -0,0 +1,26 @@
+[id="search-modules"]
+
+= How to search Modules
```

Comment:
@emarcusRH - How about making this into an assembly? You have the basis for several procedures, which should each be in their own module, and the introduction you have here sounds great as an introduction to a use case.